pkg/blocks/context.go

```diff
@@ -35,6 +35,7 @@ type TTPExecutionConfig struct {
 	NoCleanup           bool
 	CleanupDelaySeconds uint
 	Args                map[string]any
+	Targets             map[string]any
```

Comment:
This probably shouldn't be `map[string]any` - `Args` is that data type because it stores strings, ints, bools, etc - but all your stuff is strings